src/backend/core/roles/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+# Create your models here.
+
+
+class ManagerPermission(models.Model):
+    name = models.CharField(max_length=200, primary_key=True)
+    model = models.CharField(max_length=200)
+    description = models.CharField(max_length=200)
+    type = models.CharField(max_length=200)
+
+    def __str__(self):
+        return self.name
+
+
+class ManagerGroup(models.Model):
```

Comment:
So a group could be for example `Editor` with some predefined permissions, right?

Reply:
Exactly